aw_server/api.py

```diff
@@ -20,6 +23,18 @@
 logger = logging.getLogger(__name__)
 
 
+def get_device_id() -> str:
+    path = Path(get_data_dir("aw-server")) / "device_id"
```

Comment:
Same question here as aw-server-rust, why not put it in the normal config?

Reply:
I explained in the aw-server-rust PR.